adoc/admin-security-certificates.adoc

```diff
@@ -217,38 +268,56 @@ sudo cat /etc/kubernetes/admin.conf
 
 . Log in to the master node and regenerate the certificates:
 +
-* Replace the oidc-dex secret:
-+
-[source,bash]
-----
-cd /etc/kubernetes/pki
-----
+* Replace the oidc-dex server certificate:
 +
-Sign the oidc-dex server certificate with the root CA `ca.crt/ca.key` and SAN
-as <CONTROL_PLANE_IP_ADDRESS/FQDN>. Then update the {kube} cluster secret data
+Sign the oidc-dex server certificate with the CA `ca.crt/ca.key` and SAN
+as origin was. Then update the {kube} cluster secret data
```

Comment:
I don't quite understand this "as origin was" bit. :) What is origin in this case?